geode/src/main/java/org/apache/calcite/adapter/geode/rel/GeodeSchema.java

```diff
@@ -41,13 +41,13 @@
   private final List<String> regionNames;
   private ImmutableMap<String, Table> tableMap;
 
-  GeodeSchema(String locatorHost, int locatorPort,
+  public GeodeSchema(String locatorHost, int locatorPort,
```

Comment:
If there is already a public constructor with `GemFireCache`  why do we need another with `locatorHost /  locatorPort` ? Is it possible to have just one constructor ?